<?xml version="1.0"?>

<launch>

  <!-- Start Gazebo with wg world running in (max) realtime -->
  <include file="$(find hector_gazebo_worlds)/launch/rolling_landscape_120m.launch"/>

  <!-- Spawn simulated quadrotor uav -->
  <include file="$(find orca_afq)/launch/spawn_two_quadrotors.launch" >
    <arg name="model" value="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro"/>
  </include>

  <!-- xterm -e gdb -e (doubleslash)args -->
  <node pkg="orca_afq" type="simulation" name="simulation" required="true"/>

  <!-- Start rviz visualization with preset config -->
  <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find hector_quadrotor_demo)/rviz_cfg/outdoor_flight.rviz"/> -->

</launch>
